ui/.eslintrc.js

```diff
@@ -68,6 +68,9 @@ module.exports = {
       // test files
       files: ['tests/**/*-test.{js,ts}'],
       extends: ['plugin:qunit/recommended'],
+      rules: {
+        'qunit/require-expect': 'off',
```

Comment:
this rule would have made us remove `assert.expect` in places where it really does help ensure that certain assertions get called (eg in mirage server requests). So I've turned it off and we can revisit in the future if we want to update our tests and turn it on. https://github.com/platinumazure/eslint-plugin-qunit/blob/main/docs/rules/require-expect.md